app/models/category.rb

```diff
@@ -0,0 +1,6 @@
+class Category < ApplicationRecord
+  has_many :courses
+  
+  validates :name, presence: true, uniqueness: { case_sensitive: false }, length: { maximum: 100 }
```

Comment:
what does mean by it `uniqueness: { case_sensitive: false }` here ?

Reply:
Apu, this validation checks that the value of the `name` attribute must be unique in the database, but case is ignored when checking for uniqueness.